### DISCUSSION TITLE
latest update 3.6.2 doesn't output sound on windows 11

### DISCUSSION THREAD

**Discussion by Terru03** [pear-devs/pear-desktop#2511] [Q&A] [answered]
I checked every setting in Windows and all of them are set to my default speakers, they work fine with the Chrome web version of yt music, but when I try to play a song through the YouTube-Music-Web-Setup-3.6.2.exe it doesn't work. Yesterday worked fine, without any problems.

Any way I could fix this?

**Comment by Lockszmith-GH** [comment]
Right click the speaker icon in the tray area, and open the Volume Mixer - you should have per-app control over there.

**Reply by Terru03** [reply]
I tried every speaker that shows up in there, checked if they were on mute or not. I only have one speaker connected and set as default. I installed 3.5.3.0 and works without any problems

**Reply by Lockszmith-GH** [reply, marked as answer]
I'm on Windows 11, and the latest version works, so some more investigation will be needed to understand your specific case.

NO doubt frustrating. Let's wait for a maintainer to respond, maybe something changed in the last version that affects sound output.

May I suggest to make sure everything is updated (check out **UniGetUI** and **SDIO Origin** to help with that), and then report here which windows build you have installed, which locale (language) and any other detail you can think of.

**Reply by JellyBrick** [reply]
We didn't  change anything. Maybe chromium bug?

**Reply by Lockszmith-GH** [reply]
Anything OP can clear to make sure nothing is cached?

**Reply by Terru03** [reply]
I tried uninstalling and reinstalling. I am on windows 11 23h2. I have updated everything using UniGetUI and now it works on 3.6.2
